dist/angular-validation.js

```diff
@@ -346,6 +347,10 @@
                     var messageElem,
                         messageToShow = validMessage || $validationProvider.getDefaultMsg(validation).error;
 
+                    if(!ctrl.$error[ctrl.$name] && ctrl.$dirty && ctrl.$invalid){
```

Comment:
Thank´s @nmccready! I come from visual studio and nuget world...i will solve that